src/components/header/Header.js

```diff
@@ -30,21 +35,22 @@ function Header() {
             <a href="#opensource">Open Source</a>
           </li>
           <li>
-            <a href="#projects">Projects</a>
+            <a href="#blogs">Blogs</a>
           </li>
           <li>
-            <a href="#achievements">Achievements</a>
+            <a href="#talks">Talks</a>
           </li>
           <li>
-            <a href="#blogs">Blogs</a>
+            <a href="#achievements">Achievements</a>
           </li>
           <li>
-            <a href="#talks">Talks</a>
+            <a href="#contact">Contact Me</a>
           </li>
           <li>
-            <a href="#contact">Contact Me</a>
```

Comment:
**Question**: Why the order was changed? It has to in chronological order right? #116

Reply:
You are right I guess we should change the order to which the components are listed